config/migrations/20230306103000-remove-themesofsubcase.sparql

```diff
@@ -0,0 +1,17 @@
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX dct: <http://purl.org/dc/terms/>
+
+DELETE {
+  GRAPH ?g {
+    ?nieuwsbericht a ext:Nieuwsbericht ;
+                   dct:issued ?issued ;
+                   ext:themesOfSubcase ?theme .
+  }
+}
```

Comment:
Also `dct:issued` was mentioned in the Jira ticket and in queries but not specifically "flagged for removal".
On export to the Themis stack we insert that relation in the themis DB.
WIth the upcoming changes that themis stack will be integrated into our stack I'm pretty sure we want to keep that relation since it will become "active data".

Reply:
fixed